update-gemfile-lock.sh

```diff
@@ -3,4 +3,4 @@
 docker run -it --rm \
   -v $(pwd):/app \
   $(grep ^FROM docker/Dockerfile |cut -d ' ' -f2) \
-  /bin/bash -c 'apt-get update -qq && apt-get install -y --no-install-recommends make && cd /app && gem install bundler && bundle install --jobs 3 && bundle update; echo "LOCK_FILE_UPDATE_EXIT_CODE=$?"'
+  /bin/bash -c 'cd /app && gem install bundler && bundle lock --update; echo "LOCK_FILE_UPDATE_EXIT_CODE=$?"'
```

Comment:
The stock container that is used as part of this needs `make` to be installed before bundler is run.

Reply:
I tested it and it worked fine, can you share somewhere that it requires `make`?